index.html

```diff
@@ -0,0 +1,451 @@
+                  src="assets/images/rect.svg"
+                  alt="rectangle"
+                />
+                <h2>Frameworks</h2>
+              </li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+
+          <div class="third">
+            <ul class="objects">
+              <li class="rubric skill">
+                <img
+                  class="circle"
+                  src="assets/images/circle.svg"
+                  alt="circle"
+                />
+                <h2>Skills</h2>
+              </li>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section class="footer">
+        <div class="introduction">
+          <p>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </div>
+
+        <div class="form">
+          <form action="https://formspree.io/f/mqknkkgn" method="POST">
+            <ul>
+              <li class="name">
+                <label for="name"></label>
+                <input
+                  class="text"
+                  type="text"
+                  id="name"
+                  name="full_name"
+                  placeholder="Full name"
+                  minlength="3"
+                  maxlength="30"
+                  required
+                />
+                <img
+                  class="break"
+                  src="assets/images/Container.svg"
+                  alt="line"
+                />
+              </li>
+
+              <li class="mail">
+                <label for="mail"></label>
+                <input
+                  class="text"
+                  type="email"
+                  id="mail"
+                  name="user_email"
+                  placeholder="Email address"
+                  required
+                />
+                <img
+                  class="break"
+                  src="assets/images/Container.svg"
+                  alt="line"
+                />
+              </li>
+
+              <li class="blog">
+                <label for="msg"></label>
+                <textarea
+                  class="area"
+                  id="msg"
+                  name="user_message"
+                  rows="10"
+                  cols="50"
+                  placeholder="Enter text here"
+                  minlength="7"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li class="btn">
+                <button class="touch" type="submit">Get in touch</button>
+              </li>
+            </ul>
+          </form>
```

Comment:
Again, I implement here this design, due to project requirements. Here is a screenshot: 
![footer](https://user-images.githubusercontent.com/52704856/145531649-70a5ffaa-d67a-4823-bf4f-c9536897ff6b.png)

Reply:
I understand. You really did a great job here. Please watch the background color. Then there are no other boders on the inputs but a bottom border.

Reply:
Thank you for your response, Henry. But here you put a new suggestion, but don't answer my comment. 

In my previous comment, I just have written that I can't follow your suggestion - "to make your form look as close as possible to the design" because of project requirement, which screenshot I have attached above, and which says that I should implement here an approach from the mobile version, not the desktop version.

So, please, kindly explain should I follow project requirements, or your requirements, when they do not coincide? 

I am just a beginner and get into a mess in situations like this.

Reply:
Ok. I agree with you on the requirement to have a single input element for both the first and last name. What I wanted to say is this; your form has a white background color and there seem to be a black rectangular border around it. The template doesn't have that.

Just remove the white background and leave only the bottom border to make it look the same.

Reply:
Done.